src/NHibernate/Driver/DB2CoreDriver.cs

```diff
@@ -1,27 +1,16 @@
-using System.Data.Common;
-using NHibernate.SqlTypes;
+using System;
 
 namespace NHibernate.Driver
 {
 	/// <summary>
 	/// A NHibernate Driver for using the IBM.Data.DB2.Core DataProvider.
 	/// </summary>
-	public class DB2CoreDriver : DB2DriverBase
-	{		
+	// Since v5.6
+	[Obsolete("Please use DB2NetDriver")]
+	public class DB2CoreDriver : DB2NetDriver
```

Comment:
Obsoleting a driver is not user source noticeable in most cases, since it is usually setup through string value configuration.

We should issue a log warning to increase the likeliness it might be seen.